src/DataStructures/Tensor/EagerMath/Magnitude.hpp

```diff
@@ -175,14 +191,30 @@ struct Sqrt : db::ComputeTag {
 
 /// \ingroup DataBoxTagsGroup
 /// \ingroup DataStructuresGroup
-/// The L2 norm of a tensor
+/// The point-wise L2 norm of a tensor
 ///
-/// \snippet Test_Magnitude.cpp sqrt_name
-template <typename Tag, bool Accumulate = false>
+/// \snippet Test_Magnitude.cpp pointwisel2norm_name
+template <typename Tag>
+struct PointwiseL2Norm : db::ComputeTag {
+  static std::string name() noexcept {
+    return "PointwiseL2Norm(" + db::tag_name<Tag>() + ")";
+  }
+  static constexpr Scalar<DataVector> (*function)(const db::item_type<Tag>&) =
```

Comment:
can you make this `auto` now that the functions have different names?

Reply:
Compiler doesn't have enough information to deduce the template parameter `DataType` automatically